### PR TITLE
Exchange rates

### PR DESCRIPTION
**Pull request by mrsmkl**
Should now use the exchange rates from oracles, and everything should be priced in cUSD.

### CODE REVIEW

File: apps/explorer/lib/explorer/chain/celo_account.ex [hunk 1]
```diff
@@ -107,12 +107,21 @@ defmodule Explorer.Chain.CeloAccount do
   @attestation_issuer_selected "0xaf7f470b643316cf44c1f2898328a075e7602945b4f8584f48ba4ad2d8a2ea9d"
   @attestation_completed "0x414ff2c18c092697c4b8de49f515ac44f8bebc19b24553cf58ace913a6ac639d"
 
+  @median_updated "0x01f3db74cdcb3b158f2144fb78c5ab54e9e8a8c09d3d3b7713050cdb6b6bcb97"
```
Comment (jcortejoso) [inline apps/explorer/lib/explorer/chain/celo_account.ex]: Are the event addresses always the same?

**Comment by mrsmkl** [comment]
This should fix https://github.com/celo-org/celo-monorepo/issues/1000

File: apps/explorer/lib/explorer/chain/celo_account.ex (hunk 1 above)
Comment (mrsmkl) [reply apps/explorer/lib/explorer/chain/celo_account.ex]: The event topics are computed from the event signature, so if they change, their parsing has to be done differently, too.

Comment (jcortejoso) [reply apps/explorer/lib/explorer/chain/celo_account.ex]: 👍

**Pull request merged**